packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2472,18 +2468,14 @@ describe('plugin-meetings', () => {
             mediaSettings: {},
           });
 
-          assert.calledOnceWithExactly(rtcMetricsCtor, webex, meeting.id, meeting.correlationId);
-
           // check that rtcMetrics was passed to Media.createMediaConnection
           assert.calledOnce(Media.createMediaConnection);
           assert.calledWith(
             Media.createMediaConnection,
             true,
             meeting.getMediaConnectionDebugId(),
             meeting.id,
-            sinon.match({
-              rtcMetrics: fakeRtcMetrics,
-            })
+            sinon.match.hasNested('rtcMetrics.webex', webex)
```

Comment:
Here we can't mock the constructor because of the named import so we don't get the property `default`. So checking if the `rtcMetrics` is present and has property `webex` with same mock instance.